### PR TITLE
feat: added product page

### PR DESCRIPTION
**Pull request by amirnathani9**
### Made Product Page
- added filter sidebar
- added products

## Gif
productpage

## Preview Link
https://deploy-preview-4--watchesworld.netlify.app/pages/product.html

### CODE REVIEW

File: style/product.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.product-page-container{
+    width: 80vw;
+}
+
+.list-item-container{
+    background-color: transparent;
+    border-radius: 0.5rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
```
Comment (yashpatel1797) [inline style/product.css:16]: Use shorthand properties to give value.

**Review by yashpatel1797 — APPROVED** [review]
design and code looks good to merge.

File: style/product.css (hunk 1 above)
Comment (amirnathani9) [reply style/product.css:16]: padding comes from the component library here the x-axis padding is already there I have to change only the top and bottom. If i use shorthand then left and right will become zero

**Review by HarshnaNHaswani — APPROVED** [review]
Good to go, just use radio buttons for price sort and rating sort

**Review by zabihshaik — APPROVED** [review]
Everything looks good. UI looks clean.

**Comment by rahul5564** [comment]
cool

**Comment by rahul5564** [comment]
cool

**Pull request merged**